packages/web/src/components/chat-listener/ChatListener.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react'
+
+import { Status, accountSelectors, chatActions } from '@audius/common'
+import { useDispatch, useSelector } from 'react-redux'
+
+const { connect, disconnect, fetchMoreChats, fetchUnreadMessagesCount } =
+  chatActions
+const { getAccountStatus } = accountSelectors
+
+export const ChatListener = () => {
+  const dispatch = useDispatch()
+  const accountStatus = useSelector(getAccountStatus)
+  // Connect to chats websockets and prefetch chats
+  useEffect(() => {
+    if (accountStatus === Status.SUCCESS) {
+      dispatch(connect())
```

Comment:
I'm assuming it's safe for these three actions to run simultaneously without waiting for any to complete?

Reply:
Yep!